04_library_management_system.rb

```diff
@@ -77,7 +95,7 @@ def show_library_status
     puts "Name of Book      Issue Status"
     puts "-----------------------------------------------"
     @book_list.each_with_index do |book, i|
-      issue_status = (book.issued_to.nil?() || book.status == "Available" ? book.status : book.issued_to.name)
+      issue_status = (book.issued_to.nil? || book.status == "Available" ? book.status : book.issued_to.name)
       puts (i + 1).to_s + ". " + book.name + "      " + issue_status
```

Comment:
Prefer String interpolation 
```suggestion
      puts "#{i + 1} . #{book.name}  #{issue_status}"
```